lib/canvas_link_migrator/link_resolver.rb

```diff
@@ -248,9 +248,11 @@ def resolve_media_data(node, rel_path)
       elsif rel_path&.match(/\/media_attachments_iframe\/\d+/)
         # media attachment from another course or something
         rel_path
-      elsif node["data-media-id"].present?
-        file_id, uuid = @migration_id_converter.convert_attachment_media_id(node["data-media-id"])
```

Comment:
You lost the `uuid =` thing that I added recently.  I changed the media_attachment_iframe_url to use a file_id and a UUID and got rid of the lookup_attachment_by_media_id method, and this no longer matches.

Though I do see that you added that method back in in here, it's not added back in in Canvas (and should not be, since it'll add UUIDs to places that we don't want them in Canvas).  The commit to change the method in Canvas hasn't been merged, though :( https://gerrit.instructure.com/c/canvas-lms/+/349361/1/app/models/importers/db_migration_query_service.rb

Reply:
Thanks, yeah this was caused by rebasing after your changes and not being sure exactly how things should have resolved. I think I've fixed those issues.